packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -333,15 +331,17 @@ export const buildCompiledComponent = (
       ? classNameProp.value.expression
       : classNameProp.value;
 
+    const values: t.Expression[] = classNames
+      .map((className) => t.stringLiteral(className) as t.Expression)
+      .concat(classNameExpression as t.Expression);
```

Comment:
nit: hmm 🤔 2 typecastings. Is it possible we can do it one place?

Reply:
will check